dagmc/dagnav.py

```diff
@@ -420,53 +446,54 @@ def groups(self) -> list[Group]:
         """Get list of groups containing this volume."""
         return [group for group in self.model.groups.values() if self in group]
 
+    def _material_group(self):
+        for group in self.groups:
+            if self in group and group.name.startswith("mat:"):
```

Comment:
A couple of updates here. 

- The `Volume.groups` property already asserts that the volume is contained in the gruops, no need to recheck it here. 
- Double check me on this maybe, but I believe that our group name properties aren't required to be in a specific order, so as long as the keyword `mat:` appears _somewhere_ in the group name it indicates a material assignment group.

```suggestion
            if 'mat:' in group.name:
```